pkg/scheduler/capabilities/volumebinding/volume_binding.go

```diff
@@ -343,13 +343,6 @@ func (pl *VolumeBinding) PreFilter(ctx context.Context, state *framework.CycleSt
 		status.AppendReason("pod has unbound immediate PersistentVolumeClaims")
 		return nil, status
 	}
-	// Attempt to reduce down the number of nodes to consider in subsequent scheduling stages if pod has bound claims.
```

Comment:
Why is this removed and is it synced from k8s？

Reply:
Yes. ref：